uxarray/grid/grid.py

```diff
@@ -2237,15 +2245,23 @@ def to_linecollection(
         **kwargs,
     ):
         """Constructs a ``matplotlib.collections.LineCollection``` consisting
-        of lines representing the edges of the current ``Grid``
+        of lines representing the edges of the unstructured grid.
 
         Parameters
         ----------
         **kwargs: dict
-            Key word arguments to pass into the construction of a PolyCollection
+            Key word arguments to pass into the construction of a LineCollection
         """
         import cartopy.crs as ccrs
 
+        if kwargs.get("projection", False):
+            warn(
+                "Explicitly providing a projection when construction a LolyCollection has been deprecated. Please provide a projection when setting up"
```

Comment:
```suggestion
                "Explicitly providing a projection when constructing a LineCollection has been deprecated. Please provide a projection when setting up"
```

Reply:
In addition to above, the last sentence reads like incomplete, how about something like "You provide a projection later, e.g. when setting up your plot with the LineCollection."?

Reply:
Great suggestion.